scripts/zkai_proof_pressure_slope_table_gate.py

```
@@ -0,0 +1,642 @@
+            row_id="d128_to_d256_h2_seq32_width_axis",
+            axis="width",
+            from_profile_id="d128_two_head_seq32",
+            to_profile_id="d256_two_head_seq32",
+            outcome="CAUTION_WIDTH_AXIS_SAVING_WEAKENS_AND_TIMING_IS_NOT_A_SPEED_WIN",
+        ),
+    ]
+
+
+def validate_rows(rows: list[dict[str, Any]]) -> None:
+    if len(rows) != 8:
+        raise ProofPressureSlopeTableError("slope row count drift")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)